scripts/001-binutils-PPU.sh

```diff
@@ -34,18 +36,11 @@ fi
 cd ${BINUTILS}/build-ppu
 
 ## Configure the build.
-../configure --prefix="$PS3DEV/ppu" --target="powerpc64-ps3-elf" \
-    --disable-nls \
-    --disable-shared \
-    --disable-debug \
-    --disable-dependency-tracking \
-    --disable-werror \
-    --enable-64-bit-bfd \
-    --with-gcc \
-    --with-gnu-as \
-    --with-gnu-ld
+CFLAGS="-O3" CXXFLAGS="-O3" CFLAGS_FOR_TARGET="-O3" CXXFLAGS_FOR_TARGET="-O3" BOOT_CFLAGS="-O3" GOCFLAGS_FOR_TARGET="-O3" \
+../configure --prefix="$PS3DEV/ppu" --target="powerpc64-ps3-elf" --disable-nls --disable-shared --disable-debug --disable-dependency-tracking \
+             --disable-werror --enable-64-bit-bfd --with-gcc --with-gnu-as --with-gnu-ld --with-cpu="cell" --with-tune="cell" \
```

Comment:
Hm, I feel that it was easier to read when the options were spread out on more lines, but I don't have a strong opinion here.